hathor/transaction/storage/transaction_storage.py

```diff
@@ -1116,3 +1123,10 @@ def get_blocks_before(self, hash_bytes: bytes, num_blocks: int = 100) -> List[Bl
                     used.add(parent_hash)
                     pending_visits.append(parent_hash)
         return result
+
+    def flush(self) -> None:
+        """Flushes the storage. It's called during shutdown of the node, for instance.
+
+           Should be implemented by storages that provide some kind of in-memory cache
+        """
```

Comment:
I think we can remove the docstring here, so it's not duplicated from the base interface method.